include/lsst/jointcal/Gtransfo.h

```diff
@@ -139,10 +139,18 @@ class Gtransfo {
 //! allows 'stream << Transfo;' (by calling gtransfo.dump(stream)).
 std::ostream &operator<<(std::ostream &stream, const Gtransfo &gtransfo);
 
-//! Returns a pointer to a composition. if left->reduceCompo(right) return NULL, builds a GtransfoComposition
-//! and returns it. deletion of returned value to be done by caller
-
-std::unique_ptr<Gtransfo> gtransfoCompose(const Gtransfo *left, const Gtransfo *right);
+/**
+ * Returns a pointer to a composition of gtransfos, representing `left(right())`.
```

Comment:
Allow me to again register my dissatisfaction with the name `gtransfo`...

Reply:
I'm open to a new name (possibly `AstrometryTransfo` to match the newer `PhotometryTransfo`?) but that should happen on a different ticket, I think: it'd be a big change to manage.

Reply:
Why would you want to call anything a "Transfo"????????????

Reply:
As I said, I'm not renaming the object on this ticket.